defaults/defaults.go

```diff
@@ -29,4 +29,8 @@ const (
 	// DefaultSnapshotterNSLabel defines the namespace label to check for the
 	// default snapshotter
 	DefaultSnapshotterNSLabel = "containerd.io/defaults/snapshotter"
+	// DefaultWindowSize defines the default window size for stream
+	DefaultWindowSize = 2 << 20
+	// DefaultConnWindowSize defines the default window size for a connection
+	DefaultConnWindowSize = 2 << 19
```

Comment:
How do you pick the numbers? Does gRPC's default too small/large for containerd?

Reply:
According to the error reported in the [issue6190](https://github.com/containerd/containerd/issues/6190): `received 76220-bytes data exceeding the limit 65535 bytes: unknown`, In addition, I saw a similar problem & a fix from another project: [https://github.com/earthly/buildkit/pull/42/files](https://github.com/earthly/buildkit/pull/42/files), I referred it

Reply:
Could you please reflect this in the comments?

Reply:
sure, Done